### PR TITLE
Start Svalboard-specific firmware features documentation

### PR DESCRIPTION
**Pull request by vnegnev**
Added a Markdown page in a `docs` folder, with a few screenshots. Most sections are currently empty, this is just establishing a place to add feature documentation later.

As new Svalboard-specific firmware features are added, it will be helpful to future users if the firmware developers add some feature documentation to this page (or new pages could be created and linked to).

In principle the contents of the `docs` folder could later be shifted to a GitHub wiki or similar, however this is far in the future and doesn't offer any advantages yet.

### CODE REVIEW

File: keyboards/svalboard/docs/automouse_allowed_keys.png
Comment (ilc) [inline keyboards/svalboard/docs/automouse_allowed_keys.png:1]: Please in-line this as code, so it is easier to update.

File: keyboards/svalboard/readme.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Svalboard - Datahand Forever!
 
 Keyboard Maintainer: [morganvenable](https://github.com/svalboard/vial-qmk) [@_claussen] on discord
-Hardware Supported: RP2040, Svalboard Alpha 
+Hardware Supported: RP2040, Svalboard Alpha
```
Comment (ilc) [inline keyboards/svalboard/readme.md]: Add, Svalboard Lightly.

**Review by ilc — CHANGES_REQUESTED** [review]
Comments in-line,

File: keyboards/svalboard/docs/automouse_allowed_keys.png
Comment (vnegnev) [reply keyboards/svalboard/docs/automouse_allowed_keys.png:1]: Done

File: keyboards/svalboard/readme.md (hunk 1 above)
Comment (vnegnev) [reply keyboards/svalboard/readme.md]: Done

**Comment by vnegnev** [comment]
I've made the requested changes and rebased off the latest `vial` commit.

**Comment by ilc** [comment]
LGTM.

**Comment by morganvenable** [comment]
Lgtm

**Review by morganvenable — APPROVED** [review]
Approve

**Review by ilc — APPROVED** [review]
LGTM.

**Pull request merged**